onnxruntime/core/providers/cpu/rnn/rnn_helpers.cc

```diff
@@ -573,7 +573,8 @@ void gru_reset_gate_tanh(const float* ps1, float* ps2, float* pd, int c, float a
     float q = x2 * beta_6 + beta_4;
     q = x2 * q + beta_2;
     q = x2 * q + beta_0;
-    pd[i] = ps1[i] * p / q;
+    ps2[i] = p / q;
```

Comment:
>ps2 [](http://example.com/codeflow?start=4&length=3)

just to confirm, once update ps2, any impact the the caller when using ps2 after this function completes?